lib/services/keyVault/lib/keyVault.js

```diff
@@ -327,9 +327,10 @@ KeyVaultClient.prototype.getKey = function (keyIdentifier, options, callback) {
  * @param {string} keyName The name of the key
  * 
  * @param {object} [options] Optional Parameters.
- * 
- * @param {number} [options.maxresults] Maximum number of results to return.
- * 
+ *
+ * @param {number} [options.maxresults] Maximum number of results to return in
+ * a page. If not specified the service will return up to 25 results.
```

Comment:
then you need to update the swagger spec and regenerate the clients. Basically follow the pipeline.